src/opencmiss/zincwidgets/materialeditorwidget.py

```diff
@@ -193,17 +193,6 @@ def setMaterials(self, materials):
 
         material_module = self._zincContext.getMaterialmodule()
         self._material_model.setMaterialModule(material_module)
-        # Sadly not possible.
-        # self._materialmodulenotifier = material_module.createMaterialmodulenotifier()
-        # self._materialmodulenotifier.setCallback(self._materialmoduleCallback)
-
-    def _materialmoduleCallback(self, materialModuleEvent):
-        """
-        Material module does not have a notifier.
-        """
-        changeSummary = materialModuleEvent.getSummarySpectrumChangeFlags()
-        if 0 != (changeSummary & (Material.CHANGE_FLAG_IDENTIFIER | Material.CHANGE_FLAG_ADD | Material.CHANGE_FLAG_REMOVE)):
-            self._material_model.reset()
```

Comment:
I thought this function was intended to fix #19. It was marked as "not possible" and was never called, so I moved it to materialchooserwidget.py and removed it here. I can add it back if it can be used for other purposes in the future.